src/database/postgres.ts

```diff
@@ -205,6 +205,12 @@ export class DatabaseWrapperPostgres {
 		return list.filter(p => p.tournament.status === TournamentStatus.PREPARING).map(p => this.wrap(p.tournament));
 	}
 
+	async getServerTournaments(serverId: string): Promise<DatabaseTournament[]> {
+		const list = await ChallongeTournament.find({ where: { owningDiscordServer: serverId } });
```

Comment:
I'm not sure if I need to take extra steps here to filter for non-finished tournaments.

Reply:
Yes, it's needed. Given the additional baggage here I think I should extract the events (messageCreate, guildMemberRemove) in a small refactor before we commit to more of it.